populate_index.sh

```diff
@@ -15,11 +15,23 @@ curl -u admin:admin\
   --url http://localhost:8675/try_cloaked_search \
   --header 'Content-Type: application/json' \
   --data '{
-	"mappings": {
-		"properties": {
-			"_icl_encrypted_source": { "enabled": false }
-		}
-	}
+  "mappings": {
+    "dynamic_templates": [
+          {
+            "protected_fields": {
+              "match_mapping_type": "string",
+              "match": "_icl_p_*",
+              "mapping": {
+                "type": "text"
+              }
+            }
+          }
+        ],
+    "properties": {
+      "_icl_encrypted_source": { "enabled": false },
+      "_icl_search_key_id" : { "type": "keyword" }
+    }
+  }
```

Comment:
The spacing seems very odd.
```suggestion
  "mappings": {
    "dynamic_templates": [
      {
        "protected_fields": {
          "match_mapping_type": "string",
          "match": "_icl_p_*",
          "mapping": {
            "type": "text"
          }
        }
      }
    ],
    "properties": {
      "_icl_encrypted_source": { "enabled": false },
      "_icl_search_key_id": { "type": "keyword" }
    }
  }

```

Reply:
Thanks. I think opening this in vim at one point made some stuff weird.